relayer/chain.go

```diff
@@ -286,6 +286,10 @@ func (src *Chain) GetAddress() (sdk.AccAddress, error) {
 		return src.address, nil
 	}
 
+	// Set sdk config to use custom Bech32 account prefix
+	sdkConf := sdk.GetConfig()
```

Comment:
there is a new way to do this. Can you replace these lines with:

```
defer src.UseSDKContext()()
```

Reply:
For some reason, this is not working as expected. Debugging . . .

Reply:
For some reason, this is not working as expected. Debugging . . .